MREUnityRuntime/MREUnityRuntimeLib/Core/Components/AnimationComponent.cs

```diff
@@ -97,6 +153,24 @@ void AddQuaternionPatch(Type type, string propertyName, float time, QuaternionPa
 
                 void AddTransformPatch(float time, TransformPatch value)
                 {
+                    // Work around a Unity bug/feature where all position components must be specified
```

Comment:
Not necessary for rotation?

Reply:
A well-formed (complete) rotation quaternion is enforced above.